guillotina_amqp/worker.py

```diff
@@ -215,8 +215,17 @@ def num_running(self):
 
     @staticmethod
     def measure_job_duration(job, final_status):
+        dotted_name = job.data['func']
```

Comment:
It depends on the type of job though right? Only object tasks will be in this format.

Could we work with both types still and be a little more defensive in the logic here?

Reply:
totally right! :+1: just fixed it in the last commit